10up/classifai
# Add utility to migrate settings

### Description of the Change
PR adds migration script to migrate existing configured settings to new feature first settings introduced in classifAI v3.


Closes #670

### How to test the Change
1. Clone this repo
2. Checkout to `trunk` branch, and build the plugin files
3. Activate the plugin and configure all of the AI providers.
4. Checkout to this PR branch. `upkeep/670`
5. Verify that you see the notice about ClassifiAI settings migrated and changes in v3
6. Go to Tools > ClassifAI and make sure all settings migrated properly.
7. Validate each features are working properly.


### Changelog Entry

> Added - Tool to migrate feature settings from versions < 3.0.0


### Credits

Props @dkotter @Sidsector9 @iamdharmesh 


### Checklist:
